Pod/Classes/AppViewController.swift

```diff
@@ -25,6 +25,14 @@ import UIKit
 
 open class AppViewController: UIViewController {
     
+    open override var shouldAutorotate: Bool {
+        return installedViewController?.shouldAutorotate ?? true
```

Comment:
I'm just thinking that maybe instead of defaulting to `true` we should default to `super.shouldAutorotate`.